ROADMAP.md

```diff
@@ -91,6 +91,7 @@
 - [Image translation](https://github.com/machinetranslate/machinetranslate.org/issues/92)
 - [Input correction](https://github.com/machinetranslate/machinetranslate.org/issues/93)
 - [Transliteration](https://github.com/machinetranslate/machinetranslate.org/issues/94)
+* Website translation
```

Comment:
I'd say as an input mode, it's "Text translation", and should be *first*!  Thanks for catching that!

"Browser translation" is a great application or integration, we definitely need that!

"Website translation" is more like an application or content type.  It's really different if we mean the browser doing it vs. a company localizing its own website.